components/availability/src/Availability.svelte

```diff
@@ -234,7 +235,7 @@
     // TODO - Use a library to calculate the diff between props so that
     // only updated props are reassigned instead of rebuilding the whole object.
     if (
-      (!$$props.event_to_hover ||
+      ($$props.event_to_hover === undefined ||
```

Comment:
Can `$$props.event_to_hover` have a value other than `undefined` like `null`?

Reply:
That was the issue, the booking component sets this to `null` when an option is blurred which would trigger a re-fetch here.